src/components/ForecastDay.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import moment from 'moment';
```

Comment:
you can just `import {unix} from 'moment'`

Reply:
@lucas-aragno